modules/creating-instance-oci.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-instance-oci_{context}"]
+= Creating and instance on OCI
+
```

Comment:
Not sure, but instances already created after we run Stack create + Apply. 
Need to clarify this situation

Reply:
Thanks, @yobshans . @adriengentil , is this true? Is this docs [section](https://69901--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_oci/installing-oci-assisted-installer#creating-instance-oci_installing-oci-assisted-installer) needed?

Reply:
Thanks, @yobshans .

I deleted this section and removed the following steps:

![Screenshot from 2024-02-14 11-50-29](https://github.com/openshift/openshift-docs/assets/57954076/0fcf2a16-3621-4696-b07a-ee10e6499ce8)

Reply:
Adrien confirmed that the " Open the custom manifest template" step is still required, so I added it as step 6 in the "Downloading manifest files and deployment resources" section.